### PR TITLE
Add missing space

### PR DESCRIPTION
**Pull request by tomjaguarpaw**
## What?

Fix missing space (typo)

### CODE REVIEW

File: resources/content/pages/board-nominations/board-nominations-en.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ about_content:
...
     * Members will have fixed terms, to ensure a steady turnover of members.  There is a balance here: it’s a pity to lose strong, well-qualified members too quickly.  The details remain to be settled, but will be something like: three or four year terms, but with the possibility of being renewed once, and the possibility of returning after a gap.
-    * Terms will be staggered so that a similar number of members reach the end of their term at regular intervals.  “Regular intervals” might mean annually or every two years; again there is a balance between providing a regular “way in” and the overheads of nomination, selection, and onboarding.To bootstrap this process the Interim Board may invite some members to serve for shorter initial terms.
+    * Terms will be staggered so that a similar number of members reach the end of their term at regular intervals.  “Regular intervals” might mean annually or every two years; again there is a balance between providing a regular “way in” and the overheads of nomination, selection, and onboarding. To bootstrap this process the Interim Board may invite some members to serve for shorter initial terms.
```
Comment (endgame) [inline resources/content/pages/board-nominations/board-nominations-en.md:34]: Looks like most sentences in this file use two spaces after a full stop. Maybe add one more?

Comment (tomjaguarpaw) [reply resources/content/pages/board-nominations/board-nominations-en.md:34]: Hi endgame, thanks for the feedback. In order to minimise churn I would like to restrict updates to this PR to those changes requested by maintainers of the repo. Please confirm whether you are a maintainer and if so I shall get on the change straight away.

Comment (endgame) [reply resources/content/pages/board-nominations/board-nominations-en.md:34]: I am not a maintainer.

Comment (emilypi) [reply resources/content/pages/board-nominations/board-nominations-en.md:34]: Those actually might be an artifact of a bad copy-pasta, which isn't something we'd like to replicate. I agree with @tomjaguarpaw here @endgame - let's reduce the churn, and we'll have updates for the rest accordingly.

**Review by emilypi — APPROVED** [review]

**Pull request merged**